src/services/email.service.js

```diff
@@ -0,0 +1,67 @@
+'use strict';
+
+const nodemailer = require('nodemailer');
+
+const transporter = nodemailer.createTransport({
+  host: process.env.SMTP_HOST,
+  port: process.env.SMTP_PORT,
+  secure: false,
+  auth: {
+    user: process.env.SMTP_USER,
+    pass: process.env.SMTP_PASSWORD,
+  },
```

Comment:
you can additionally add env validation with `zod` or `yup`:
https://catalins.tech/validate-environment-variables-with-zod/

Reply:
In general, I wonder why this is not on the course. Because validation is the basis of working with forms